HISTORY.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version next
 
+## Version 1.4.0
```

Comment:
Can you not add v1.4.0 yet? Just keep it under next. Then when we release the package we update the version and merge `dev` into `master`. See https://github.com/netuitive-community-packages/netuitive-packages-rabbitmq/commit/16d7ab24d642fc6f0b6cdece24b62692d12fa746 for an example version bump directly on `dev`.